csrc/scheduler/reduction_utils.h

```diff
@@ -70,6 +71,7 @@ NVF_API void propagateParallelization(
     const std::vector<TensorView*>& reduction_tvs,
     const std::vector<TensorView*>& cached_inputs,
     const std::vector<std::pair<TensorView*, TensorView*>>& cached_outputs,
+    const std::vector<TensorView*>& smem_persistent_buffer_consumers = {},
```

Comment:
This function is getting difficult to understand and manage. Probably it has too many parameters. Please clean it up. Also, it's unacceptable to have no code comment for functions with this many parameters.

Reply:
cleaned in https://github.com/NVIDIA/Fuser/pull/3196